.storybook/webpack.config.js

```diff
@@ -1,19 +1,13 @@
-const { rules, loaders, plugins, stats } = require('webpack-atoms')
+const { rules } = require('webpack-atoms')
 
 const browsers = ['last 2 versions', 'ie >= 10']
 
-module.exports = function(config) {
-  return {
-    ...config,
-    module: {
-      rules: [
-        rules.js(),
-        rules.fonts(),
-        rules.images(),
-        rules.css(),
-        rules.less({ browsers }),
-      ],
-    },
-    plugins: [...config.plugins],
-  }
+module.exports = function({ config }) {
+  config.module.rules.push(rules.js())
+  config.module.rules.push(rules.fonts())
+  config.module.rules.push(rules.images())
+  config.module.rules.push(rules.css())
+  config.module.rules.push(rules.less({ browsers }))
```

Comment:
this will now use the default rules, which i don't think we want...

Reply:
I've pushed an update to completely override the 'rules'. They did have a rule (from the `@storybook/react` package) already for the js compile, as well as some bits for handling media, but this new config more closely mirrors what you were doing before.